.github/workflows/ci.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     timeout-minutes: 60
     strategy:
       matrix:
-        rust: [ nightly-2022-08-08 ]
+        rust: [ nightly-2023-02-02 ]
```

Comment:
I suggest we remove this matrix, and add a env instead in L13, 

Then we can just use it like
```
rustup toolchain install ${{ env.RUST_VERSION }} --profile minimal

``